lib/problem1.rb

```diff
@@ -0,0 +1,3 @@
+def problem1(values)
+  values.reduce(0) { |sum, value| value % 2 == 0 ? sum + value : sum }
```

Comment:
Ruby's tag line is 'developer pleasure', it has this `?` methods can you search the docs and find one that will be a good fit for this line of code?

Reply:
I've replaced the modulo operator use with the `even?` method